easygoogle/src/main/java/pub/devrel/easygoogle/gac/SignIn.java

```diff
@@ -42,6 +42,11 @@
  * (when possible) however this does not grant any authorization to use other Google APIs on
  * behalf of the user. For more information visit:
  * https://developers.google.com/identity/sign-in/android/
+ *
+ * When registering OAuth clients for Google Sign-In, Web server OAuth client registration is needed
+ * with android by requesting an ID token in your GoogleSignInOptions, and supplying the web client
+ * ID for your server. For more information visit:
+ * http://android-developers.blogspot.sg/2016/03/registering-oauth-clients-for-google.html
```

Comment:
No problem with linking to a blog, but can we use https:// and the `.com` link just to pick the most common?
https://android-developers.blogspot.com/2016/03/registering-oauth-clients-for-google.html